updater/cmd/updater/main.go

```diff
@@ -68,12 +70,34 @@ func main() {
 	}
 
 	rb := rollback.NewRollbacker(logger, installDir)
+	// Stop the service before backing up the install directory;
+	// We want to stop as early as possible so that we don't hit the collector's timeout
+	// while it waits to be shutdown.
+	service := service.NewService(logger, installDir)
```

Comment:
i'm not going to complain too much, but maybe to something to think about when we give this whole thing a once over. it would have been nice if we could have just had a method to call on the installer to do the stopping, instead of copying "a bunch" (i know its really not that much) of logic from it to here

Reply:
Yeah, good call out, definitely should be a part of the larger refactor.